packages/calypso-e2e/src/lib/flows/start-import-flow.ts

```diff
@@ -114,6 +116,13 @@ export class StartImportFlow {
 		await this.page.waitForSelector( selectors.wpContentOnlyContinueButton );
 	}
 
+	/**
+	 * Validates that we've landed on the WordPress plans page.
+	 */
+	async validateWordPressPlansPage(): Promise< void > {
+		await this.page.waitForSelector( selectors.wpPlansContentOnlyOptionButton );
```

Comment:
**question:non-blocking**- without knowing the screen that this is referring to, I can't say for certain but the description sounds like it's checking that the user has been redirected to the`/plans` endpoint.

If so, can I suggest usage of the `PlansPage.validateActivePlan` or `PlansPage.validateActiveTab` in order to validate that the expected action took place _without_ introducing a new single-use method?

Alternatively, checking for `page.waitForURL` in the test is always a possibility.

Reply:
Actually, it's a pre-migration page with plans information in it. I've renamed the function a little, and I hope it's clearer now.